java8/Java8.g4

```diff
@@ -348,7 +348,8 @@ fieldModifier
 	;
 
 variableDeclaratorList
-	:	variableDeclarator (',' variableDeclarator)*
+	:	variableDeclarator
+	|	variableDeclaratorList ',' variableDeclarator
```

Comment:
In this case, your rule has a worse performance. It does not reduce the number of rules in comparison of replacing a sequence of rules with a single left-recursive rule.